OracleWebLogic/samples/12213-webtier-apache/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Example of Apache Web Server with WebLogic plugin for load balancing WebLogic on Docker Containers
+#
+# Copyright (c) 2016-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip
+# Download the Generic installer from http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html and save it in this same folder.
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t 12213-apache . 
+# or
+# RUN:
+#      $ sh build.sh
+#
+# For Oracle developers, use build_internal.sh, which set up HTTP/HTTPS proxies.
+# RUN:
+#      $ sh build_internal.sh
+#
+# Note if needed, pull the latest httpd image and tag it as: httpd:2.4
+#
+FROM httpd:2.4
```

Comment:
This has to be changed to `FROM oraclelinux:7-slim`

Reply:
Of course, we'll have a CMD if we change to use oraclelinux:7-slim as the base. I'll make the change. Thanks.

Reply:
Avi

The image we are trying to create is the  WebLogic Server Apache HTTP Server Proxy Plugin, we need to extend the Apache HTTP Server Project since we need  the scripts and functionality already included in the image.  When we do a yum -y install httpd inside the container to install Apache HTTPd 2.4 we do not have the same directory or code.  What is the reason we need to base this on OL slim?